flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -99,7 +107,7 @@ class CDUAocOfpData {
         }
 
         async function setTargetCargo(numberOfPax, simbriefCargo) {
-            const bagWeight = numberOfPax * 20;
+            const bagWeight = numberOfPax * BAG_WEIGHT;
             const maxLoadInCargoHold = 9435; // from flight_model.cfg
             const maxTotalPayload = 21800; // from flight_model.cfg
             let loadableCargoWeight = undefined;
```

Comment:
Does something need to be changed in the loadable cargo weight loop to allow for the case that someone can pick a per passenger bag weight high enough that the bag weight alone can exceed the max cargo load hold, or when combined with the passenger weight, the max total payload?

Reply:
When loading manually: the per pax/bag values are still the same as before, which wouldn't allow it to exceed
When loading via Simbrief using custom per pax/bag values: the OFP generation takes care of excess load as mentioned by Derek from SB- " But beware: In case the aircraft is in an overweight situation and payload has to be removed to respect the MTOW or MZFW, cargo will be the first thing that gets removed. First, <freight_added> gets removed, and if that isn't enough, baggage weight goes next. So in these cases it is possible for (<pax_count> * <bag_weight>) to not match up with the <cargo> value. If you need to, you can perform (<cargo> / <bag_weight>) to figure out approximately how many bags actually remain on board. "

Reply:
> But Simbrief doesn't know the max cargo hold load, only the MTOW or MZFW.

Yes, but Simbrief also added something which restricts the max per baggage weight. If trying to set unrealistic values, this is what is displayed on the airframe page
![image](https://user-images.githubusercontent.com/29005005/153746673-7916c09d-9763-43d7-b07e-b86526c6a0a5.png)

Reply:
@donstim I thought about it again, and this seems like a cleaner way. We restrict max weight of baggage and freight in cargo hold to 9435KGs and not max payload (21800KGs). Have pushed the changes accordingly